lib/ansible/module_utils/vmware.py

```diff
@@ -12,9 +12,10 @@
 import time
 from random import randint
 
+# requests is required for exception handling of the ConnectionError
+import requests
```

Comment:
We should never just let `ImportError` fly through to the user.  If you want to separate it, it should go into something like:

```
try:
    import requests
    HAS_REQUESTS = True
except ImportError:
    HAS_REQUESTS = False
```

Then later use `HAS_REQUESTS` to provide a user friendly error message.